stage/stage.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+<body>
+	Stage ok!
```

Comment:
you should not have stage code commited, the code should be build for stage, may need to do a bit more research on evimrents

Reply:
stage no longer committed